src/pal/tests/palsuite/c_runtime/__iscsym/test1/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ add_executable(paltest_iscsym_test1
 add_dependencies(paltest_iscsym_test1 coreclrpal)
 
 target_link_libraries(paltest_iscsym_test1
-  pthread
```

Comment:
So I would rather recommend creating a cmake variable COMMON_TEST_LIBRARIES in the src/pal/tests/CMakeLists.txt that would contain pthread, m and coreclrpal for non-Android and just the last two for Android. Then all tests would have ${COMMON_TEST_LIBRARIES} here instead of the three.